cmd/polaris/audit.go

```diff
@@ -202,7 +203,9 @@ func outputAudit(auditData validator.AuditData, outputFile, outputURL, outputFor
 			} else {
 				req.Header.Set("Content-Type", "text/plain")
 			}
-			client := &http.Client{}
+			client := &http.Client{Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			}}
```

Comment:
@fengshunli this is super dangerous--at most we should make this an option and turn it off by default.